!SESSION 2018-01-04 17:53:23.953 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_92
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=zh_CN
Command-line arguments:  -os win32 -ws win32 -arch x86

This is a continuation of log file D:\WorkSpace\MyCode\GE_EFM8LB12F64E-B-QFN32_YJ\.metadata\.bak_0.log
Created Time: 2018-01-04 17:54:06.459

!ENTRY org.eclipse.equinox.p2.transport.ecf 2 0 2018-01-04 17:54:06.461
!MESSAGE Connection to https://devtools.silabs.com/studio/v4/updates/p2.index failed on devtools.silabs.com. Retry attempt 0 started
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:131)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.performConnect(HttpClientRetrieveFileTransfer.java:1077)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.access$0(HttpClientRetrieveFileTransfer.java:1068)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer$1.performFileTransfer(HttpClientRetrieveFileTransfer.java:1064)
	at org.eclipse.ecf.filetransfer.FileTransferJob.run(FileTransferJob.java:73)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.installer.core 4 0 2018-01-04 17:54:06.787
!MESSAGE Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
!STACK 1
org.eclipse.equinox.p2.core.ProvisionException: Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
	at org.eclipse.equinox.internal.p2.repository.CacheManager.createCache(CacheManager.java:243)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.getLocalFile(SimpleMetadataRepositoryFactory.java:66)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.load(SimpleMetadataRepositoryFactory.java:88)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.factoryLoad(MetadataRepositoryManager.java:57)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:768)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:668)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.basicRefreshRepository(AbstractRepositoryManager.java:203)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.refreshRepository(MetadataRepositoryManager.java:100)
	at com.silabs.ss.platform.internal.api.installer.core.SLSP2UpdateSites.ensureStudioReposLoaded(SLSP2UpdateSites.java:620)
	at com.silabs.ss.platform.internal.installer.core.scripting.SLSInstallerModule.updateInstallationMetadata(SLSInstallerModule.java:470)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:208)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	... 1 more
!SUBENTRY 1 org.eclipse.equinox.p2.transport.ecf 4 1006 2018-01-04 17:54:06.787
!MESSAGE Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 17:54:06.848
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.checkForUpdates(AssetService.java:428)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:235)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 17:54:06.853
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.checkForUpdates(AssetService.java:428)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:235)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-04 17:54:06.855
!MESSAGE Assets checkForUpdates failed due to not online

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2018-01-04 18:00:33.870
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:00:38.431
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:00:38.435
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:00:38.447
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:00:38.450
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 1 0 2018-01-04 18:00:38.452
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!STACK 0
java.lang.Exception: /efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json is not currently available offline.  Please make sure you have Internet access.
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:378)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.framework.debugger.core 4 0 2018-01-04 18:00:38.454
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Asset error while getting device descriptor(s) for agent
	at com.silabs.ss.platform.api.rcp.core.StatusReporter.newCoreException(StatusReporter.java:239)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:63)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	... 14 more
!SUBENTRY 1 com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-04 18:00:38.455
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
!SUBENTRY 2 com.silabs.ss.platform.content.core 2 0 2018-01-04 18:00:38.455
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-04 18:00:38.456
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2018-01-04 18:04:02.762
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:04:06.724
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:04:06.726
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:04:06.731
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-04 18:04:06.733
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 1 0 2018-01-04 18:04:06.735
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!STACK 0
java.lang.Exception: /efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json is not currently available offline.  Please make sure you have Internet access.
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:378)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.framework.debugger.core 4 0 2018-01-04 18:04:06.737
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Asset error while getting device descriptor(s) for agent
	at com.silabs.ss.platform.api.rcp.core.StatusReporter.newCoreException(StatusReporter.java:239)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:63)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	... 14 more
!SUBENTRY 1 com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-04 18:04:06.737
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
!SUBENTRY 2 com.silabs.ss.platform.content.core 2 0 2018-01-04 18:04:06.738
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-04 18:04:06.742
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!SESSION 2018-01-17 15:02:25.466 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_92
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86, WS=win32, NL=zh_CN
Command-line arguments:  -os win32 -ws win32 -arch x86

!ENTRY org.eclipse.core.resources 2 10035 2018-01-17 15:02:27.730
!MESSAGE The workspace exited with unsaved changes in the previous session; refreshing workspace to recover changes.

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:02:40.278
!MESSAGE Failed to load assets file: C:\SiliconLabs\SimplicityStudio\v4\configuration\studio\assets\assets_aem.assets3
!STACK 0
org.eclipse.emf.ecore.resource.impl.ResourceSetImpl$1DiagnosticWrappedException: org.xml.sax.SAXParseExceptionpublicId: file:/C:/SiliconLabs/SimplicityStudio/v4/configuration/studio/assets/assets_aem.assets3; systemId: file:/C:/SiliconLabs/SimplicityStudio/v4/configuration/studio/assets/assets_aem.assets3; lineNumber: 1; columnNumber: 50; 在 publicId 和 systemId 之间需要有空格。
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.handleDemandLoadException(ResourceSetImpl.java:319)
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.demandLoadHelper(ResourceSetImpl.java:278)
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.getResource(ResourceSetImpl.java:406)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.loadRemoteAssets(AssetService.java:688)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.loadAssetsFiles(AssetService.java:795)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.init(AssetService.java:608)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.<init>(AssetService.java:155)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at org.eclipse.equinox.internal.ds.model.ServiceComponent.createInstance(ServiceComponent.java:493)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.createInstance(ServiceComponentProp.java:270)
	at org.eclipse.equinox.internal.ds.model.ServiceComponentProp.build(ServiceComponentProp.java:331)
	at org.eclipse.equinox.internal.ds.InstanceProcess.buildComponent(InstanceProcess.java:620)
	at org.eclipse.equinox.internal.ds.ServiceReg.getService(ServiceReg.java:53)
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse$1.run(ServiceFactoryUse.java:212)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.factoryGetService(ServiceFactoryUse.java:210)
	at org.eclipse.osgi.internal.serviceregistry.ServiceFactoryUse.getService(ServiceFactoryUse.java:111)
	at org.eclipse.osgi.internal.serviceregistry.ServiceConsumer$2.getService(ServiceConsumer.java:45)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistrationImpl.getService(ServiceRegistrationImpl.java:496)
	at org.eclipse.osgi.internal.serviceregistry.ServiceRegistry.getService(ServiceRegistry.java:461)
	at org.eclipse.osgi.internal.framework.BundleContextImpl.getService(BundleContextImpl.java:619)
	at org.eclipse.e4.core.internal.contexts.osgi.EclipseContextOSGi.lookup(EclipseContextOSGi.java:85)
	at org.eclipse.e4.core.internal.contexts.osgi.EclipseContextOSGi.containsKey(EclipseContextOSGi.java:61)
	at org.eclipse.e4.core.internal.contexts.EclipseContext.containsKey(EclipseContext.java:141)
	at org.eclipse.e4.core.internal.contexts.ContextObjectSupplier$ContextInjectionListener.update(ContextObjectSupplier.java:69)
	at org.eclipse.e4.core.internal.contexts.TrackableComputationExt.update(TrackableComputationExt.java:111)
	at org.eclipse.e4.core.internal.contexts.EclipseContext.runAndTrack(EclipseContext.java:322)
	at org.eclipse.e4.core.internal.contexts.ContextObjectSupplier.get(ContextObjectSupplier.java:170)
	at org.eclipse.e4.core.internal.di.InjectorImpl.resolveArgs(InjectorImpl.java:477)
	at org.eclipse.e4.core.internal.di.InjectorImpl.resolveRequestorArgs(InjectorImpl.java:405)
	at org.eclipse.e4.core.internal.di.InjectorImpl.inject(InjectorImpl.java:110)
	at org.eclipse.e4.core.internal.di.InjectorImpl.inject(InjectorImpl.java:86)
	at org.eclipse.e4.core.contexts.ContextInjectionFactory.inject(ContextInjectionFactory.java:73)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.<init>(UpdateManager.java:52)
	at com.silabs.ss.platform.internal.api.installer.core.updates.IUpdateManager.<clinit>(IUpdateManager.java:21)
	at com.silabs.ss.platform.internal.installer.ui.update.StudioUpdateChecker.<init>(StudioUpdateChecker.java:81)
	at com.silabs.ss.platform.internal.installer.ui.update.StudioUpdateCheckerComponent.createService(StudioUpdateCheckerComponent.java:13)
	at org.eclipse.equinox.internal.p2.core.ProvisioningAgent.getService(ProvisioningAgent.java:72)
	at org.eclipse.equinox.internal.p2.ui.sdk.scheduler.AutomaticUpdateScheduler.<init>(AutomaticUpdateScheduler.java:76)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at org.eclipse.core.internal.registry.osgi.RegistryStrategyOSGI.createExecutableExtension(RegistryStrategyOSGI.java:184)
	at org.eclipse.core.internal.registry.ExtensionRegistry.createExecutableExtension(ExtensionRegistry.java:905)
	at org.eclipse.core.internal.registry.ConfigurationElement.createExecutableExtension(ConfigurationElement.java:243)
	at org.eclipse.core.internal.registry.ConfigurationElementHandle.createExecutableExtension(ConfigurationElementHandle.java:55)
	at org.eclipse.ui.internal.WorkbenchPlugin$1.run(WorkbenchPlugin.java:293)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:52)
	at org.eclipse.ui.internal.WorkbenchPlugin.createExtension(WorkbenchPlugin.java:288)
	at org.eclipse.ui.internal.EarlyStartupRunnable.getExecutableExtension(EarlyStartupRunnable.java:120)
	at org.eclipse.ui.internal.EarlyStartupRunnable.run(EarlyStartupRunnable.java:68)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.ui.internal.Workbench$56.run(Workbench.java:2812)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: org.xml.sax.SAXParseExceptionpublicId: file:/C:/SiliconLabs/SimplicityStudio/v4/configuration/studio/assets/assets_aem.assets3; systemId: file:/C:/SiliconLabs/SimplicityStudio/v4/configuration/studio/assets/assets_aem.assets3; lineNumber: 1; columnNumber: 50; 在 publicId 和 systemId 之间需要有空格。
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1438)
	at com.sun.org.apache.xerces.internal.impl.XMLScanner.scanExternalID(XMLScanner.java:1050)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.scanDoctypeDecl(XMLDocumentScannerImpl.java:646)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next(XMLDocumentScannerImpl.java:929)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:510)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:848)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:777)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:643)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:327)
	at org.eclipse.emf.ecore.xmi.impl.XMLLoadImpl.load(XMLLoadImpl.java:175)
	at org.eclipse.emf.ecore.xmi.impl.XMLResourceImpl.doLoad(XMLResourceImpl.java:261)
	at org.eclipse.emf.ecore.resource.impl.ResourceImpl.load(ResourceImpl.java:1518)
	at org.eclipse.emf.ecore.resource.impl.ResourceImpl.load(ResourceImpl.java:1297)
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.demandLoad(ResourceSetImpl.java:259)
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.demandLoadHelper(ResourceSetImpl.java:274)
	... 58 more

!ENTRY com.silabs.ss.platform.apack.ip 2 0 2018-01-17 15:02:40.524
!MESSAGE No network interfaces found.

!ENTRY com.silabs.ss.platform.device.core 4 0 2018-01-17 15:02:41.840
!MESSAGE Inspector installation request exception 
!STACK 0
java.lang.IllegalStateException: Preference node "https:__devtools.silabs.com_studio_v4_updates" has been removed.
	at org.eclipse.core.internal.preferences.EclipsePreferences.checkRemoved(EclipsePreferences.java:200)
	at org.eclipse.core.internal.preferences.EclipsePreferences.internalNode(EclipsePreferences.java:653)
	at org.eclipse.core.internal.preferences.EclipsePreferences.node(EclipsePreferences.java:812)
	at org.eclipse.core.internal.preferences.EclipsePreferences.internalNode(EclipsePreferences.java:676)
	at org.eclipse.core.internal.preferences.EclipsePreferences.node(EclipsePreferences.java:812)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.remember(AbstractRepositoryManager.java:883)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.addRepository(AbstractRepositoryManager.java:170)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:655)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.query(AbstractRepositoryManager.java:1167)
	at com.silabs.ss.platform.internal.api.installer.core.InstallerUtils.findFeaturesForCapabilities(InstallerUtils.java:600)
	at com.silabs.ss.platform.internal.api.installer.core.InstallerUtils.scanFeaturesProvidingMetadata(InstallerUtils.java:371)
	at com.silabs.ss.platform.internal.api.installer.core.InstallerUtils.getPackageProvidingMetadata(InstallerUtils.java:392)
	at com.silabs.ss.platform.internal.installer.core.PackageService.loadRemotePackages(PackageService.java:606)
	at com.silabs.ss.platform.internal.installer.core.PackageService.getAllAvailablePackages(PackageService.java:100)
	at com.silabs.ss.platform.internal.installer.core.PackageService.getAvailablePackages(PackageService.java:135)
	at com.silabs.ss.platform.internal.installer.core.PackageService.requestPackageTypeInstalled(PackageService.java:492)
	at com.silabs.ss.platform.api.device.core.DeviceManager.triggerInspectors(DeviceManager.java:342)
	at com.silabs.ss.platform.api.device.core.DeviceManager.addDevice(DeviceManager.java:304)
	at com.silabs.ss.platform.api.device.core.detection.DeviceDetectionManager.processDetectionInfo(DeviceDetectionManager.java:198)
	at com.silabs.ss.platform.api.device.core.detection.DeviceDetectionManager.list(DeviceDetectionManager.java:379)
	at com.silabs.ss.platform.internal.apack.device.UsbDeviceBridgeListener.list(UsbHotplugNotifier.java:130)
	at com.silabs.ss.platform.api.apack.usb.UsbManager.lambda$7(UsbManager.java:201)
	at com.silabs.ss.platform.api.rcp.core.ListenerList.fire(ListenerList.java:120)
	at com.silabs.ss.platform.api.apack.usb.UsbManager.parseAction(UsbManager.java:201)
	at com.silabs.ss.platform.api.apack.usb.UsbManager.parseAction(UsbManager.java:1)
	at com.silabs.java.utils.LineParser$InternalActionListener.parseAction(LineParser.java:93)
	at com.silabs.ss.platform.internal.apack.usb.UsbDetectorLineHandler.handle(UsbDetectorLineHandler.java:65)
	at com.silabs.ss.platform.internal.apack.usb.UsbDetectorLineHandler.handle(UsbDetectorLineHandler.java:1)
	at com.silabs.ss.platform.api.apack.core.execution.ErrorLoggerLineHandler.handle(Executioner.java:361)
	at com.silabs.ss.platform.api.apack.core.execution.ErrorLoggerLineHandler.handle(Executioner.java:1)
	at com.silabs.java.utils.LineParser.parse(LineParser.java:76)
	at com.silabs.java.utils.LineParser.parse(LineParser.java:62)
	at com.silabs.ss.platform.api.apack.core.execution.ParserThread.run(Executioner.java:323)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:02:42.172
!MESSAGE Exception comparing timestamps for: /common/Master_Software_License_Agreement.zip
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1513)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1441)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.isAssetUpdateNeeded(AssetService.java:508)
	at com.silabs.ss.platform.internal.api.installer.core.LicenseRetriever.updateLicenses(LicenseRetriever.java:284)
	at com.silabs.ss.platform.internal.api.installer.ui.CheckLicenses.needsUpdate(CheckLicenses.java:87)
	at com.silabs.ss.platform.internal.api.installer.ui.CheckLicenses.checkLicenses(CheckLicenses.java:73)
	at com.silabs.ss.platform.internal.product.AppStartupCompleteEventHandler$1.run(AppStartupCompleteEventHandler.java:82)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:02:42.175
!MESSAGE Local timestamp file does not exist: C:\SiliconLabs\SimplicityStudio\v4\offline\common\Simplicity_Studio_Terms_of_Use_and_Privacy_Statement.timestamp

!ENTRY org.eclipse.equinox.p2.transport.ecf 2 0 2018-01-17 15:02:42.331
!MESSAGE Connection to https://devtools.silabs.com/studio/v4/updates/p2.index failed on devtools.silabs.com. Retry attempt 0 started
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:131)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.performConnect(HttpClientRetrieveFileTransfer.java:1077)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.access$0(HttpClientRetrieveFileTransfer.java:1068)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer$1.performFileTransfer(HttpClientRetrieveFileTransfer.java:1064)
	at org.eclipse.ecf.filetransfer.FileTransferJob.run(FileTransferJob.java:73)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.installer.core 4 0 2018-01-17 15:02:42.668
!MESSAGE Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
!STACK 1
org.eclipse.equinox.p2.core.ProvisionException: Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
	at org.eclipse.equinox.internal.p2.repository.CacheManager.createCache(CacheManager.java:243)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.getLocalFile(SimpleMetadataRepositoryFactory.java:66)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.load(SimpleMetadataRepositoryFactory.java:88)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.factoryLoad(MetadataRepositoryManager.java:57)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:768)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:668)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.basicRefreshRepository(AbstractRepositoryManager.java:203)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.refreshRepository(MetadataRepositoryManager.java:100)
	at com.silabs.ss.platform.internal.api.installer.core.SLSP2UpdateSites.ensureStudioReposLoaded(SLSP2UpdateSites.java:620)
	at com.silabs.ss.platform.internal.installer.core.scripting.SLSInstallerModule.createSession(SLSInstallerModule.java:537)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:206)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	... 1 more
!SUBENTRY 1 org.eclipse.equinox.p2.transport.ecf 4 1006 2018-01-17 15:02:42.669
!MESSAGE Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY org.eclipse.equinox.p2.transport.ecf 2 0 2018-01-17 15:02:42.699
!MESSAGE Connection to https://devtools.silabs.com/studio/v4/updates/p2.index failed on devtools.silabs.com. Retry attempt 0 started
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:144)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:131)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.performConnect(HttpClientRetrieveFileTransfer.java:1077)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer.access$0(HttpClientRetrieveFileTransfer.java:1068)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientRetrieveFileTransfer$1.performFileTransfer(HttpClientRetrieveFileTransfer.java:1064)
	at org.eclipse.ecf.filetransfer.FileTransferJob.run(FileTransferJob.java:73)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.installer.core 4 0 2018-01-17 15:02:42.965
!MESSAGE Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
!STACK 1
org.eclipse.equinox.p2.core.ProvisionException: Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
	at org.eclipse.equinox.internal.p2.repository.CacheManager.createCache(CacheManager.java:243)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.getLocalFile(SimpleMetadataRepositoryFactory.java:66)
	at org.eclipse.equinox.internal.p2.metadata.repository.SimpleMetadataRepositoryFactory.load(SimpleMetadataRepositoryFactory.java:88)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.factoryLoad(MetadataRepositoryManager.java:57)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:768)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.loadRepository(AbstractRepositoryManager.java:668)
	at org.eclipse.equinox.internal.p2.repository.helpers.AbstractRepositoryManager.basicRefreshRepository(AbstractRepositoryManager.java:203)
	at org.eclipse.equinox.internal.p2.metadata.repository.MetadataRepositoryManager.refreshRepository(MetadataRepositoryManager.java:100)
	at com.silabs.ss.platform.internal.api.installer.core.SLSP2UpdateSites.ensureStudioReposLoaded(SLSP2UpdateSites.java:620)
	at com.silabs.ss.platform.internal.installer.core.scripting.SLSInstallerModule.updateInstallationMetadata(SLSInstallerModule.java:470)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:208)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	... 1 more
!SUBENTRY 1 org.eclipse.equinox.p2.transport.ecf 4 1006 2018-01-17 15:02:42.966
!MESSAGE Unknown Host: https://devtools.silabs.com/studio/v4/updates/content.xml
!STACK 0
java.net.UnknownHostException: devtools.silabs.com
	at java.net.InetAddress.getAllByName0(InetAddress.java:1280)
	at java.net.InetAddress.getAllByName(InetAddress.java:1192)
	at java.net.InetAddress.getAllByName(InetAddress.java:1126)
	at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:44)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.resolveHostname(DefaultClientConnectionOperator.java:259)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:159)
	at org.apache.http.impl.conn.ManagedClientConnectionImpl.open(ManagedClientConnectionImpl.java:304)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:611)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:446)
	at org.apache.http.impl.client.AbstractHttpClient.doExecute(AbstractHttpClient.java:863)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
	at org.eclipse.ecf.provider.filetransfer.httpclient4.HttpClientFileSystemBrowser.runRequest(HttpClientFileSystemBrowser.java:259)
	at org.eclipse.ecf.provider.filetransfer.browse.AbstractFileSystemBrowser$DirectoryJob.run(AbstractFileSystemBrowser.java:69)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:02:42.995
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.checkForUpdates(AssetService.java:428)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:235)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:02:42.999
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.checkForUpdates(AssetService.java:428)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.updatePackageMetadata(UpdateManager.java:235)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.ensureUpdatesFetched(UpdateManager.java:283)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager.access$5(UpdateManager.java:274)
	at com.silabs.ss.platform.internal.installer.core.updates.UpdateManager$1.run(UpdateManager.java:166)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-17 15:02:43.000
!MESSAGE Assets checkForUpdates failed due to not online

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2018-01-17 15:03:34.513
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:03:38.230
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:03:38.233
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:03:38.241
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:03:38.244
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 1 0 2018-01-17 15:03:38.245
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!STACK 0
java.lang.Exception: /efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json is not currently available offline.  Please make sure you have Internet access.
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:378)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.framework.debugger.core 4 0 2018-01-17 15:03:38.247
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Asset error while getting device descriptor(s) for agent
	at com.silabs.ss.platform.api.rcp.core.StatusReporter.newCoreException(StatusReporter.java:239)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:63)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	... 14 more
!SUBENTRY 1 com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-17 15:03:38.248
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
!SUBENTRY 2 com.silabs.ss.platform.content.core 2 0 2018-01-17 15:03:38.248
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-17 15:03:38.250
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2018-01-17 15:05:38.350
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:05:42.183
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:05:42.191
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:05:42.195
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:05:42.197
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 1 0 2018-01-17 15:05:42.199
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!STACK 0
java.lang.Exception: /efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json is not currently available offline.  Please make sure you have Internet access.
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:378)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.framework.debugger.core 4 0 2018-01-17 15:05:42.201
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Asset error while getting device descriptor(s) for agent
	at com.silabs.ss.platform.api.rcp.core.StatusReporter.newCoreException(StatusReporter.java:239)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:63)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	... 14 more
!SUBENTRY 1 com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-17 15:05:42.202
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
!SUBENTRY 2 com.silabs.ss.platform.content.core 2 0 2018-01-17 15:05:42.202
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-17 15:05:42.204
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2018-01-17 15:08:37.414
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:08:41.483
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:08:41.486
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:08:41.490
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:08:41.492
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 1 0 2018-01-17 15:08:41.493
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!STACK 0
java.lang.Exception: /efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json is not currently available offline.  Please make sure you have Internet access.
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:378)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-17 15:08:41.494
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.core 4 0 2018-01-17 15:08:41.496
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Asset error while getting device descriptor(s) for agent
	at com.silabs.ss.platform.api.rcp.core.StatusReporter.newCoreException(StatusReporter.java:239)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:63)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	... 14 more
!SUBENTRY 1 com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-17 15:08:41.496
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
!SUBENTRY 2 com.silabs.ss.platform.content.core 2 0 2018-01-17 15:08:41.497
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 1 0 2018-01-17 15:09:36.435
!MESSAGE Launching SiAgent (Silicon Labs TCF Agent)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:09:40.332
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:09:40.344
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:376)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:09:40.348
!MESSAGE Failed to connect, verify on line, due to exception https://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.security.ssl.SSLSocketImpl.connect(SSLSocketImpl.java:668)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
	at sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:367)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:191)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:177)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.connect(HttpsURLConnectionImpl.java:153)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:944)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 4 0 2018-01-17 15:09:40.350
!MESSAGE Failed to connect, verify on line, due to exception http://developer.silabs.com/studio/v4/assets/
!STACK 0
java.net.UnknownHostException: developer.silabs.com
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:184)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at sun.net.NetworkClient.doConnect(NetworkClient.java:175)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
	at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
	at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
	at sun.net.www.http.HttpClient.New(HttpClient.java:308)
	at sun.net.www.http.HttpClient.New(HttpClient.java:326)
	at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1169)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1105)
	at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:999)
	at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:933)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.online(AssetService.java:959)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.access$0(AssetService.java:935)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:325)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 1 0 2018-01-17 15:09:40.351
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
!STACK 0
java.lang.Exception: /efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json is not currently available offline.  Please make sure you have Internet access.
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:378)
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService$2.run(AssetService.java:323)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)

!ENTRY com.silabs.ss.platform.content.core 2 0 2018-01-17 15:09:40.352
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.core 4 0 2018-01-17 15:09:40.354
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Asset error while getting device descriptor(s) for agent
	at com.silabs.ss.platform.api.rcp.core.StatusReporter.newCoreException(StatusReporter.java:239)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:63)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	... 14 more
!SUBENTRY 1 com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-17 15:09:40.354
!MESSAGE Asset error while getting device descriptor(s) for agent
!STACK 1
org.eclipse.core.runtime.CoreException: Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json
	at com.silabs.ss.platform.internal.api.content.core.asset.AssetService.resolveAsset(AssetService.java:340)
	at com.silabs.ss.framework.debugger.api.tcf.agent.SiAgentUtils.resolveDeviceDescriptors(SiAgentUtils.java:49)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence.ensurePartDescriptor(SLSLaunchSequence.java:640)
	at com.silabs.ss.framework.debugger.api.core.SLSLaunchSequence$2.execute(SLSLaunchSequence.java:100)
	at org.eclipse.cdt.dsf.concurrent.Sequence.executeStep(Sequence.java:459)
	at org.eclipse.cdt.dsf.concurrent.Sequence.access$2(Sequence.java:373)
	at org.eclipse.cdt.dsf.concurrent.Sequence$2.handleSuccess(Sequence.java:420)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor.handleCompleted(RequestMonitor.java:385)
	at org.eclipse.cdt.dsf.concurrent.RequestMonitor$2.run(RequestMonitor.java:312)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
!SUBENTRY 2 com.silabs.ss.platform.content.core 2 0 2018-01-17 15:09:40.354
!MESSAGE Unable to download asset://efm8/debug/lb1/EFM8LB1_B/EFM8LB12F64E-B-QFN32_debug.json

!ENTRY com.silabs.ss.framework.debugger.tcf.core 4 0 2018-01-17 15:10:32.373
!MESSAGE TCF task failed
!STACK 0
java.io.IOException: Command [C 412 DebugAdapter disconnect "root.000440059121"] aborted
	at org.eclipse.tcf.core.AbstractChannel$9.run(AbstractChannel.java:753)
	at org.eclipse.tcf.EventQueue.run(EventQueue.java:95)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:209)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:284)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at java.io.FilterInputStream.read(FilterInputStream.java:107)
	at org.eclipse.tcf.core.ChannelTCP.get(ChannelTCP.java:162)
	at org.eclipse.tcf.core.StreamChannel.read(StreamChannel.java:84)
	at org.eclipse.tcf.core.AbstractChannel$1.run(AbstractChannel.java:254)

!ENTRY org.eclipse.tcf 4 0 2018-01-17 15:10:32.377
!MESSAGE Cannot close channel streams
!STACK 0
java.net.SocketException: Socket closed
	at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:116)
	at java.net.SocketOutputStream.write(SocketOutputStream.java:153)
	at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
	at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140)
	at java.io.FilterOutputStream.close(FilterOutputStream.java:158)
	at org.eclipse.tcf.core.ChannelTCP.stop(ChannelTCP.java:200)
	at org.eclipse.tcf.core.AbstractChannel$8.run(AbstractChannel.java:716)

!ENTRY com.silabs.ss.framework.debugger.tcf.agent 4 0 2018-01-17 15:10:32.379
!MESSAGE Lost connection to agent
!STACK 0
java.net.SocketException: Connection reset
	at java.net.SocketInputStream.read(SocketInputStream.java:209)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:284)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at java.io.FilterInputStream.read(FilterInputStream.java:107)
	at org.eclipse.tcf.core.ChannelTCP.get(ChannelTCP.java:162)
	at org.eclipse.tcf.core.StreamChannel.read(StreamChannel.java:84)
	at org.eclipse.tcf.core.AbstractChannel$1.run(AbstractChannel.java:254)

!ENTRY org.eclipse.cdt.dsf 4 -1 2018-01-17 15:10:32.392
!MESSAGE Invocation exception when calling a service event handler method
!STACK 0
java.lang.reflect.InvocationTargetException
	at sun.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:531)
	at org.eclipse.cdt.dsf.service.DsfSession.access$2(DsfSession.java:475)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:398)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.util.concurrent.RejectedExecutionException: Display org.eclipse.swt.widgets.Display@1feb1ef is disposed
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:85)
	at org.eclipse.cdt.dsf.ui.viewmodel.AbstractVMAdapter.handleEvent(AbstractVMAdapter.java:260)
	at org.eclipse.cdt.dsf.ui.viewmodel.datamodel.AbstractDMVMAdapter.eventDispatched(AbstractDMVMAdapter.java:103)
	... 13 more
Caused by: org.eclipse.swt.SWTException: Device is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4491)
	at org.eclipse.swt.SWT.error(SWT.java:4406)
	at org.eclipse.swt.SWT.error(SWT.java:4377)
	at org.eclipse.swt.widgets.Display.error(Display.java:1258)
	at org.eclipse.swt.widgets.Display.asyncExec(Display.java:709)
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:77)
	... 15 more
Root exception:
java.util.concurrent.RejectedExecutionException: Display org.eclipse.swt.widgets.Display@1feb1ef is disposed
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:85)
	at org.eclipse.cdt.dsf.ui.viewmodel.AbstractVMAdapter.handleEvent(AbstractVMAdapter.java:260)
	at org.eclipse.cdt.dsf.ui.viewmodel.datamodel.AbstractDMVMAdapter.eventDispatched(AbstractDMVMAdapter.java:103)
	at sun.reflect.GeneratedMethodAccessor59.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.cdt.dsf.service.DsfSession.doDispatchEvent(DsfSession.java:531)
	at org.eclipse.cdt.dsf.service.DsfSession.access$2(DsfSession.java:475)
	at org.eclipse.cdt.dsf.service.DsfSession$3.run(DsfSession.java:398)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.eclipse.swt.SWTException: Device is disposed
	at org.eclipse.swt.SWT.error(SWT.java:4491)
	at org.eclipse.swt.SWT.error(SWT.java:4406)
	at org.eclipse.swt.SWT.error(SWT.java:4377)
	at org.eclipse.swt.widgets.Display.error(Display.java:1258)
	at org.eclipse.swt.widgets.Display.asyncExec(Display.java:709)
	at org.eclipse.cdt.dsf.ui.concurrent.SimpleDisplayExecutor.execute(SimpleDisplayExecutor.java:77)
	... 15 more
